topics/shell/directories_check.md

```diff
@@ -0,0 +1,5 @@
+## Directories Check
+
+### Objectives
+
+1. write a script to Check if a directory exists.
```

Comment:
s/write a script to Check if a directory exists./Write a script to check if a directory exists./